hadoop-ozone/dist/src/main/smoketest/grpc/grpc-om-s3-metrics.robot

```diff
@@ -21,7 +21,8 @@ Library             BuiltIn
 Resource            ../commonlib.robot
 Resource            ../s3/commonawslib.robot
 Test Timeout        5 minutes
-Suite Setup         Setup s3 tests
+Suite Setup         Run Keywords    Get Security Enabled From Config
+...                 AND             Setup s3 tests
```

Comment:
I think this change is unnecessary.  `Setup s3 tests` already calls `Get Security Enabled From Config`:

https://github.com/apache/ozone/blob/6d08145fe67b7ca1df0bfc4e674169ba6bc771ce/hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot#L133-L137